package.json

```diff
@@ -374,6 +374,7 @@
 			"simplemde>codemirror": "^5.65.11",
 			"simplemde>marked": "^4.3.0",
 			"sharp": "^0.33.2",
+			"@fluid-tools/benchmark": "file:./tools/benchmark",
```

Comment:
We'll need to revert this before merging

Reply:
Tested locally. Remove it :)